DEBUGGING.md

```diff
@@ -0,0 +1,101 @@
+## Debugging
+
+Debugging is one of the requested features of language support. Currently Kotlin native compiler produces debug info compatible
+with DWARF 2 specification, that allow modern debugger tools provide basic support:
```

Comment:
"that allow modern debugger tools provide basic support" ->
"so modern debugging tools could perform following operations:"